root/user/tag.tt

```diff
@@ -1,10 +1,11 @@
 [% WRAPPER 'user/profile/layout.tt' page='tags' %]
 
 [% IF tag_in_use %]
+    <h2>[% tag_name %]</h2>
```

Comment:
This should be escaped like `[% tag_name | html %]` if you don't want arbitrary scripts executing on the page. ;)